content/en/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins.md

```diff
@@ -11,16 +11,11 @@ weight: 10
 
 <!-- overview -->
 
-{{< feature-state state="alpha" >}}
-{{< caution >}}Alpha features can change rapidly. {{< /caution >}}
-
 Network plugins in Kubernetes come in a few flavors:
 
 * CNI plugins: adhere to the appc/CNI specification, designed for interoperability.
```

Comment:
How about:
```suggestion
* CNI plugins: adhere to the [Container Network Interface](https://github.com/containernetworking/cni) (CNI)
   specification.
   * Kubernetes follows v0.4.0 of the CNI specification.
```

Reply:
_optionally_, link to https://github.com/containernetworking/cni/blob/spec-v0.4.0/SPEC.md  ?

Reply:
SGTM 👍 Thanks!